src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -5572,7 +6294,9 @@ public static class Extensions2
 {
     public static C.Enumerator GetAsyncEnumerator(this C self) => new C.Enumerator();
 }";
-            CompileAndVerify(source, parseOptions: TestOptions.RegularPreview, expectedOutput: "123");
+            var comp = CreateCompilationWithMscorlib46(source, options: TestOptions.DebugExe, parseOptions: TestOptions.RegularPreview);
+            comp.VerifyDiagnostics();
```

Comment:
`CompileAndVerify` does this check, no need to do it explicitly.

Reply:
And belo.w

---
In reply to: [443869820](https://github.com/dotnet/roslyn/pull/45160#discussion_r443869820) [](ancestors = 443869820)

Reply:
It doesn't show warnings, which I think are valuable to see and react to changes to